use crate::Error;

pub async fn handle_prediction_begin() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_prediction_progress() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_prediction_lock() -> Result<(), Error> {
    Ok(())
}

pub async fn handle_prediction_end() -> Result<(), Error> {
    Ok(())
}
